src/sas/qtgui/Utilities/DocViewWidget.py

```diff
@@ -207,8 +209,31 @@ def loadHtml(self):
         settings.setAttribute(QtWebEngineCore.QWebEngineSettings.LocalContentCanAccessFileUrls, True)
         self.webEngineViewer.load(url)
 
+        # Set widget title to include the name of the document loaded
+        self.updateTitle()
+
         # Show widget
         self.onShow()
+    
+    def updateTitle(self):
+        """
+        Set the title of the window to include the name of the document,
+        found in the first <h1> tags.
+        """
+        # Convert QUrl to pathlib path
+        try:
+            current_path = re.search(r'//(.*?)\'\)', str(self.webEngineViewer.url()))
+            current_path = current_path.group(1)
+        except:
+            self.setWindowTitle("Documentation")
+            return
```

Comment:
This can be simplified to a single try/except with no return statements.

I'm leery about your regex. What are you trying to match here? I would suggest passing the url to `pathlib.Path` class and getting either the `name` for the filename, of `absolute()` for the pull path.

You should not use bare exceptions. `except (<A>, <TupleOf>, <ExceptionsIWillLikelyThrow>):` is ideal, but `except Exception:` should be the bare minimum.

```suggestion
        try:
            current_path = re.search(r'//(.*?)\'\)', str(self.webEngineViewer.url()))
            current_path = current_path.group(1)
            self.setWindowTitle(f"Documentation—{current_path.strip()}")
        except:
            self.setWindowTitle("Documentation")
```